src/main/python/tensorframes/core.py

```diff
@@ -37,6 +37,19 @@ def _get_shape(node):
     l = node.get_shape().as_list()
     return [-1 if x is None else x for x in l]
 
+def _initialize_variable(graph, fetches, initial_variable):
+    with tf.Session(graph=graph) as sess:
```

Comment:
hmm maybe? i am not really sure how mixed clusters works, but since user can force some variable with `with tf.device('/cpu:0')` so it's really usage problem? but yeah..what's your suggestion?